src/Mod/RuleItem.cpp

```diff
@@ -1657,6 +1657,8 @@ BattleType RuleItem::getBattleType() const
  */
 BattleFuseType RuleItem::getFuseTimerType() const
 {
+	if (_fuseType == BFT_SET && Options::instantPrime)
+		return BFT_INSTANT;
```

Comment:
This will break lot of logic around grenade priming, `BFT_INSTANT` is very special type of grenade, using user options to change could break many mods as they expect normal behavior.

If you must you should use fixed grenade time, like `BFT_FIX_MIN`.

Beside there is case where item is not grenade but use fuses to some special proposes, I think correct way to handle changes like this is simply create mod that switch manually each grenade to "fixed 0" grenade.

Reply:
Whoops. I guess I should have tested more than just the act of priming them. :o
Changed it to BFT_FIX_MIN and now seems to be working as intended.